Update User Guide -Aliases and tooltips for Table widget (#8658)
geosolutions-it/MapStore2: docs/user-guide/widgets.md

<img src="../img/widgets/table-panel.jpg" class="ms-docimage"  style="max-width:450px;"/>

> The toolbar on the top of this panel is similar to the one present in [Chart section](#chart).Here the user is allowed to:

Reviewer: ```suggestion
The toolbar on the top of this panel is similar to the one present in [Chart section](#chart). Here the user is allowed to:
```

> * **Enable/Disable** the layer fields that will be displayed in the widget.

Reviewer: ```suggestion
* **Enable/Disable** the layer fields that will be displayed in the widget as columns.
```

<img src="../img/widgets/table-attribute.gif" class="ms-docimage"/>

!!!warning
    At least one field must be selected in order to move to the next configuration step.

> * Enter a **Title** for each column to be displayed in the table header in place of the *Name* of the layer field

Reviewer: ```suggestion
* Enter a **Title** for each column to be displayed as the table header in place of the *Name* of the layer field
```

<img src="../img/widgets/table-title.gif" class="ms-docimage"/>

> * Enter a **Description** for each fields to be displayed in the table through a tooltip

Reviewer: ```suggestion
* Enter a **Description** for each field to be displayed as a tooltip, visible moving the mouse on the column header. 
```

> * Enter a **Description** for each field to be displayed as a tooltip, visible moving the mouse on the column header. 

Reviewer: ```suggestion
* Enter a **Description** for each field to be displayed as a tooltip, visible moving the mouse on the column header.
```
